msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/CacheRefreshReason.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+/**
+ * Specifies the reason for fetching the access token from the identity provider when using {@link AbstractClientApplicationBase#acquireTokenSilently(SilentParameters)}
+ */
+public enum CacheRefreshReason {
```

Comment:
What about claims?

Reply:
In the latest commit I renamed the `FORCE_REFRESH` reason to `FORCE_REFRESH_OR_CLAIMS`, to better match MSAL .NET: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/7d17705533eab68d504ee2f179364922367dfba7/src/client/Microsoft.Identity.Client/Cache/CacheRefreshReason.cs#L18

Reply:
Let's pls do the right thing from the start. Separate Force_Refresh and Claims. We need to do this in .NET too, please open a bug there.

Reply:
In the latest commit these are now separate entries in the enum.

Do they also have separate telemetry values? Or is this more for developer clarity about why the token was refreshed?